app/src/main/java/com/github/se/signify/model/stats/Stats.kt

```diff
@@ -8,5 +8,6 @@ data class Stats(
     val dailyQuest: Int = 0,
     val weeklyQuest: Int = 0,
     val completedChallenge: Int = 0,
-    val createdChallenge: Int = 0
+    val createdChallenge: Int = 0,
+    val wonChallenge: Int = 0
```

Comment:
I would split these into categories (Quests, Challenges, Practice).
I'm not a fan of these variable names either. Maybe `challengesCreated` and `easyExercisesCompleted` would sound better.

You've only matched existing code here, so I won't request changes retroactively on this PR. Maybe keep this in mind for next time.

Reply:
Noted